test/ably/resthttp_test.py

```diff
@@ -135,3 +139,16 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_request_headers(self):
+        ably = AblyRest(key=test_vars["keys"][0]["key_str"],
+                        rest_host=test_vars["host"],
+                        port=test_vars["port"],
+                        tls_port=test_vars["tls_port"],
+                        tls=test_vars["tls"])
+        r = ably.http.make_request('HEAD', '/time', skip_auth=True)
+        self.assertIn('X-Ably-Version', r.request.headers)
+        self.assertEqual(r.request.headers['X-Ably-Version'], api_version)
```

Comment:
I realise this is pedantic, but our thinking with this test is that it should prevent changes have unexpected side effects.  So as a result, if someone mistakenly changed `api_version` at `https://github.com/ably/ably-python/pull/80/files#diff-78bff66cadc66de2c6ba2e48b623cb8aR29` to `api_version = 'a'` then this test would still pass.  I would prefer that we explicitly check the Ably protocol version here as any change to a protocol is rare and updating this test is a good thing.